README.md

```diff
@@ -42,6 +42,48 @@ In `package.json`
 
 * method - The method that perform for publishing library
 
+### Peer Dependencies 
```

Comment:
ชื่อหัวข้อว่า What is watchDependencies? ดีกว่ารึเปล่า

Reply:
ok